tsconfig.json

```diff
@@ -1,22 +1,20 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
-    "declaration": true,
+    "strict": true,
+    "module": "ES6",
+    "target": "ES6",
     "noImplicitAny": false,
-    "removeComments": true,
-    "moduleResolution": "node",
+    "declaration": true,
+    "moduleResolution": "Node",
```

Comment:
is this correct?

Reply:
- I think we should keep removeComments, so I disable it
- `moduleResolution`, just make it capitalize first letter. It's valid. (hint of VSCode)